src/schema/README.md

```diff
@@ -481,6 +489,102 @@ Specifically, files in the first group may have `task`, `run`, `processing`, and
 while files in the second group may not.
 Also, when files in the second group have the `acq` entity, the associated value MUST be `crosstalk`.
 
+### `sidecars/*.yaml`
+
+Sidecar files introduce the idea of "selectors" to determine whether a set of fields is added
+to the sidecar schema that will be applied to the sidecar.
+From here, the `fields` property lists will be accumulated to set requirement levels.
```

Comment:
This is confusingly worded to me. Perhaps

```suggestion
From here, the metadata fields listed in the `fields` property will be accumulated across dictionaries to set requirement levels after evaluating all selectors.
```

That's not really any better, actually.